src/user/dto/learningProfile.dto.ts

```diff
@@ -1,35 +1,51 @@
-import { IsDefined, IsNotEmpty } from 'class-validator';
+import { IsDefined, IsNotEmpty } from "class-validator";
 
-import { LearningProfile, Skill, LearningHistory } from '@prisma/client';
-import { OmitType } from '@nestjs/swagger';
-import { CareerProfileCreationDto } from './careerProfile-creation.dto';
-import { LearningProfileCreationDto } from './learningProfile-creation.dto';
+import { LearningProfile, Skill, LearningHistory } from "@prisma/client";
+import { OmitType } from "@nestjs/swagger";
```

Comment:
Is this really used? I don't see this.

Please cleanup the code before merging. Delete all gryed imports.

Reply:
I deleted OmitType, since it wasn't used in the class.